styles/style.scss

```diff
@@ -8,10 +8,12 @@ $font-path: '../../fonts';
 @import '/node_modules/@mozilla-protocol/core/protocol/css/protocol.css';
 @import '/node_modules/@mozilla-protocol/core/protocol/css/includes/lib';
 @import '/node_modules/@mozilla-protocol/core/protocol/css/components/button';
+@import '/node_modules/@mozilla-protocol/core/protocol/css/components/breadcrumb';
```

Comment:
**Nit:** Doesn't look like you're using this component anywhere? Unless you're planning on using it, I think we should remove the extra import.